Resources under VM details (#301)
openshift/openshift-origin-design: web-console/knikubevirt/vm-details/Resources/Resources.md

# VM details - Devices-and-resources

### Virtual machine

## From VM details page
![VM details - Overview tab - default](img/VM-2-0.jpg)

> Within the VM details > overview tab > overview section, a click on the ‘Devices & resources’ field will pop up the correlated modal.

Reviewer: ```suggestion
Within the VM details > overview tab > overview section, a click on the ‘Resources’ field will pop up the correlated modal.
```

![Devices & resources modal - default](img/VM-3-0.png)

> The ‘Devices & resources’ modal currently contains only the ‘workload isolation’ option. This may expand in the future. The modal is titled this way, instead of  ‘Workload isolation’, in order to align with the implementation on the pod’s side.

Reviewer: ```suggestion
The ‘Resources’ modal currently contains only the Resources Policy. This may expand in the future. The modal is titled this way, instead of  ‘Workload isolation’, in order to align with the implementation on the pod’s side.
```

A sub-text below the title mentions that the scheduler will attempt to achieve the following settings. In case it is not possible, the VM will be scheduled in the best way it can (the same thing applies for pods, where this modal has more settings).

> The user selects ‘Isolate this workload…’ checkbox.

Reviewer: This should be updated to reflect the current modal right?

Author: Yeah, I just dropped the ball with the MD...

![VM details - Overview tab - workload isolation enabled](img/VM-4-0.jpg)

> The preview on the ‘Devices & resources’ us updated to reflect the changes.

Reviewer: ```suggestion
The preview on the ‘Resources’ us updated to reflect the changes.
```
Should we call this preview or just description?

## From VM creation wizard

> ![Create VM - Advanced - Virtual hardware](img/Create-VM-4-2-00.png)

Reviewer: Can we align this with the current design seen here:
https://codepen.io/matthewcarleton/pen/NWWZMZJ
Note the indent for each section (CD-ROMs, Display) under virtual hardware. As this section grows I think it will be useful to indent so scanning is simpler.

Author: I thought PF4 dropped the indent. Totally agree we should use it

Reviewer: agreed, updated with spacing instead. 
https://codepen.io/matthewcarleton/pen/NWWZMZJ?editors=1100

> Under ‘Create VM > Advanced step > Virtual hardware, the user can now enable workload isolation. New section titles are added to separate from the Guest agent section and the Devices & resources section.

Reviewer: ```suggestion
Under ‘Create VM > Advanced step > Virtual hardware, the user can now enable workload isolation. New section titles are added to separate from the CD-ROMs and the Resources section.
```

## From pod details page
![pod details - Overview tab - default](img/Pod-2-0.jpg)

> Within the Pod details > overview tab > overview section, a click on the ‘Devices & resources’ field will pop up the correlated modal.

Reviewer: ```suggestion
Within the Pod details > overview tab > overview section, a click on the ‘Resources’ field will pop up the correlated modal.
```
